PhpUnit/AbstractExtensionTestCase.php

```diff
@@ -44,13 +45,22 @@ protected function setUp()
      * Call this method from within your test after you have (optionally) modified the ContainerBuilder for this test
      * ($this->container).
      *
-     * @param array $specificConfiguration
+     * If your extension(s) implements \Symfony\Component\DependencyInjection\Extension\PrependExtensionInterface, you may
+     * set $withPrependInvocation to TRUE to invoke prepend() method prior to load() method of your extension.
+     *
+     * @param array $configurationValues
+     * @param bool $withPrependInvocation
      */
-    protected function load(array $configurationValues = array())
+    protected function load(array $configurationValues = array(), $withPrependInvocation = false)
```

Comment:
I agree with you both. It would be standard behavior, so you might as well call it always. I *think* that "prepend" isn't used that often though, so I won't think many people will be bitten by a bug. What would you say about adding it now, for people who are working with  a "prepend" extension already, and maybe adding an issue to make it standard behavior for the next major version?